### PR TITLE
Implement LIC14

### PR DESCRIPTION
**Pull request by Internet-Person-IP**
This PR implements LIC14 with Unit Test using Testing Convention

Fixes: #7

### CODE REVIEW

**Comment by elmaxe** [comment]
The title and the description of this PR do not match

File: src/test/java/LIC14Test.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+
+  }
+
+	  @Test
+  /*
+  * It tests a Edge case
+	* Tests if LIC14 can calculate AREAS correctly close to the end the Array.
+  */
+	  public void testLIC14EdgeCase(){
```
Comment (K2017) [inline src/test/java/LIC14Test.java:89]: Indentation

**Review by K2017 — APPROVED** [review]
Some small formatting issues,

File: src/main/java/CMV.java [hunk 2]
```diff
@@ -195,7 +196,20 @@ private boolean LIC13() {
    * Computes the LIC 14 condition.
    */
   private boolean LIC14() {
-    // TODO: Implementation.
-    return false;
+    if(points.length < 5 || parameters.AREA2 <= 0) return false;
```
Comment (elmaxe) [inline src/main/java/CMV.java:199]: AREA2 < 0 according to the specification

Comment (elmaxe) [reply src/main/java/CMV.java:199]: i.e. change to `parameters.AREA2 < 0`

File: src/main/java/CMV.java [hunk 3]
```diff
@@ -195,7 +196,20 @@ private boolean LIC13() {
...
+
+    int ePts = parameters.E_PTS;
+    int fPts = parameters.F_PTS;
+    double AREA1 = parameters.AREA1;
+    double AREA2 = parameters.AREA2;
+
+    for(int Pnt = 0; Pnt < points.length - ePts - fPts; Pnt++){
+      if(Point.triangleAreaFromPoints(points[Pnt], points[Pnt + ePts], points[Pnt + ePts + fPts]) > AREA1) greaterThanA1 = true;
+      if(Point.triangleAreaFromPoints(points[Pnt], points[Pnt + ePts], points[Pnt + ePts + fPts]) < AREA2) lessThanA2 = true;
+    }
```
Comment (elmaxe) [inline src/main/java/CMV.java:212]: If ePts and fPts is zero, the triangles will be created fro the same points. You are accessing the E_PTS, not the point after it (same for F_PTS). There is also a possible index out of bounds problem.

```java
for (int Pnt = 0; Pnt < points.length - ePts - fPts - 2; Pnt++){
  if(Point.triangleAreaFromPoints(points[Pnt], points[Pnt + ePts + 1], points[Pnt + ePts + 1 + fPts + 1]) > AREA1) greaterThanA1 = true;
  if(Point.triangleAreaFromPoints(points[Pnt], points[Pnt + ePts + 1], points[Pnt + ePts + 1 + fPts + 1]) < AREA2) lessThanA2 = true;
}
```

**Review by elmaxe — CHANGES_REQUESTED** [review]

**Review by elmaxe — APPROVED** [review]
